### PR TITLE
Fix issues for iframe partner-swap

### PR DESCRIPTION
**Pull request by neikop**
- [x] Sometimes, cant get balance token on LO, Swap -> Update "fetch balances on other chain" to use rtk-query
- [x] In Cross-Chain tab, click sub tab auto navigate to default -> Check isPartnerSwap
- [x] cant edit LO -> Check if using customChainId from url params
- [x] Change context cancel 0 limit orders -> Check if using customChainId from url params
- [x] import token according to wallet, not selected chain -> Check if using customChainId from url params

### CODE REVIEW

**Review by namgold — APPROVED** [review]

File: src/state/wallet/hooks.ts [hunk 1]
```diff
@@ -101,15 +101,16 @@ export function useTokenBalancesWithLoadingIndicator(
...
 
   const isFetchOtherChain = chainId !== currentChain
 
-  const balancesCurrentChain = useTokensBalance(isFetchOtherChain ? EMPTY_ARRAY : tokens, chainId)
+  const balancesCurrentChain = useTokensBalance(isFetchOtherChain ? [] : tokens, chainId)
+
   const [balancesOtherChain, isLoadingAnotherChain] = useTokensBalanceOfAnotherChain(
     chainId,
-    isFetchOtherChain ? tokens : EMPTY_ARRAY,
+    isFetchOtherChain ? tokens : [],
```
Comment (nguyenhoaidanh) [inline src/state/wallet/hooks.ts:113]: tell us why

Comment (neikop) [reply src/state/wallet/hooks.ts:113]: Actually I don't know why we need a const like that while we can just use []

Comment (nguyenhoaidanh) [reply src/state/wallet/hooks.ts:113]: it can make a loop if pass [] directly instead of EMPTY_ARRAY
useEffect(()=>{

},[ dependencies])

Comment (nguyenhoaidanh) [reply src/state/wallet/hooks.ts:113]: because it will create a new array when every render

Comment (neikop) [reply src/state/wallet/hooks.ts:113]: Ah okay, let's me think

Comment (neikop) [reply src/state/wallet/hooks.ts]: Reverted

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**